src/main/java/com/synopsys/integration/alert/common/ProxyManager.java

```diff
@@ -56,14 +56,14 @@ public ProxyManager(final BaseConfigurationAccessor configurationAccessor) {
             return configurationAccessor.getConfigurationByDescriptorNameAndContext(SettingsDescriptor.SETTINGS_COMPONENT, ConfigContextEnum.GLOBAL)
                        .stream()
                        .findFirst();
-        } catch (AlertDatabaseConstraintException ex) {
+        } catch (final AlertDatabaseConstraintException ex) {
             logger.error("Could not find the settings configuration for proxy data", ex);
         }
         return Optional.empty();
     }
 
-    public ProxyInfo createProxyInfo() throws IllegalArgumentException {
```

Comment:
SonarQube might complain about declaring unchecked exceptions, but we often do it to avoid accidentally introducing bugs when using certain methods. I think this is one of those cases since the exception is bubbling up from `integration-rest` and it is declared as part of the method signature.